### PR TITLE
Added search planner to Apple Watch

### LINKED ISSUE #176: Add Search View to Apple Watch

### PR DESCRIPTION
**Pull request by planecore**
Closes #176, #161

### CODE REVIEW

File: ios/BetterRail.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1339,7 +1347,7 @@
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = 4;
-				WATCHOS_DEPLOYMENT_TARGET = 7.1;
+				WATCHOS_DEPLOYMENT_TARGET = 8.0;
```
Comment (guytepper) [inline ios/BetterRail.xcodeproj/project.pbxproj:1350]: Why is that?

File: ios/WatchBetterRail Extension/Views/SearchView.swift [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          }
+          .disabled((origin == nil) || (destination == nil))
+        }
+        .navigationTitle("search route")
+        .navigationBarTitleDisplayMode(.inline)
+    }
+  
+  @ViewBuilder
+  func getRoutesView() -> some View {
+    if let origin = origin, let destination = destination {
```
Comment (guytepper) [inline ios/WatchBetterRail Extension/Views/SearchView.swift:59]:
```suggestion
    if let origin, let destination {
```

**Review by guytepper — CHANGES_REQUESTED** [review]
Amazing! 
Just fix this message (rounded corners, alignment, some padding)
simulator_screenshot_6E835CD2-6579-4F8F-8B03-E2217260B1AA

File: ios/BetterRail.xcodeproj/project.pbxproj (hunk 1 above)
Comment (planecore) [reply ios/BetterRail.xcodeproj/project.pbxproj:1350]: `.navigationBarTitleDisplayMode(.inline)` requires watchOS 8+

**Review by guytepper — APPROVED** [review]

**Pull request merged**